frontends/smv_encoder.h

```diff
@@ -6,8 +6,8 @@
 #include <string>
 #include <unordered_map>
 #include "assert.h"
-#include "exceptions.h"
-#include "rts.h"
+#include "utils/exceptions.h"
+#include "core/rts.h"
 #include "smt-switch/smt.h"
 #include "smvparser.h"
```

Comment:
do you need to also add frontends to the smvparser.h?

Reply:
Actually I don't think we do, but this isn't working quite right yet. smvparser.h is a generated file, so it needs to be installed correctly. I'll leave more info in another comment.

Reply:
Okay, I updated this now.